### DISCUSSION TITLE
请问为什么编译后启动不了呢？

### DISCUSSION THREAD

**Discussion by keeno1982** [UnblockNeteaseMusic/luci-app-unblockneteasemusic#124] [Q&A] [answered]
页面日志是提示：Failed to check latest core version, please try again later.
Core Not Found, please download it before starting.

点击删除核心和更新核心好像都不行，是哪里出错了？
系统是Lede openwrt
谢谢了

**Comment by 1715173329** [comment]
命令行输入 `unm-debug` 打印调试报告

**Reply by keeno1982** [reply]
OpenWrt info:
DISTRIB_ID='OpenWrt'
DISTRIB_RELEASE='SNAPSHOT'
DISTRIB_TARGET='armvirt/64'
DISTRIB_ARCH='aarch64_cortex-a53'
DISTRIB_TAINTS='no-all override'
DISTRIB_REVISION='R22.2.2'
DISTRIB_DESCRIPTION='OpenWrt '


uclient-fetch info:
Package: uclient-fetch
Version: 2021-05-14-6a6011df-1
Depends: libc, libuclient20201210
Provides: wget
Status: install user installed
Architecture: aarch64_cortex-a53
Installed-Time: 1645873789

Package: libustream-openssl20201210
Version: 2022-01-16-868fd881-1
Depends: libc, libubox20210516, libopenssl1.1
Provides: libustream-openssl
Status: install user installed
Architecture: aarch64_cortex-a53
Installed-Time: 1645873789

Downloading 'https://api.github.com/repos/UnblockNeteaseMusic/server/commits?sha=enhanced&path=precompiled'
Connecting to 140.82.112.6:443
Connection error: Connection failed
Failed to parse json data: unexpected end of data
Failed to connect to GitHub with uclient-fetch.


Node.js info:
Package: node
Version: v14.18.3-1
Depends: libc, libstdcpp6, libopenssl1.1, zlib, libnghttp2-14, libatomic1
Status: install user installed
Architecture: aarch64_cortex-a53
Installed-Time: 1645873789

Node.js is placed at /usr/bin/node
Node.js version: v14.18.3


luci-app-unblockneteasmusic info:
Package: luci-app-unblockneteasemusic
Version: 2.12-2
Depends: libc, busybox, dnsmasq-full, ipset, jsonfilter, node, uclient-fetch
Status: install user installed
Architecture: all
Installed-Time: 1645873789

-rw-r--r--    1 root     root         285 Feb 26 20:48 /etc/config/unblockneteasemusic
-rwxr-xr-x    1 root     root        8.6K Feb 26 20:33 /etc/init.d/unblockneteasemusic

/usr/share/unblockneteasemusic:
-rwxr-xr-x    1 root     root        4.7K Feb 26 20:33 debugging.sh
-rwxr-xr-x    1 root     root         348 Feb 26 20:33 log_check.sh
-rwxr-xr-x    1 root     root        2.9K Feb 26 20:33 update.sh

config unblockneteasemusic 'config'
        option use_custom_cookie '0'
        option local_vip '0'
        option auto_update '1'
        option update_time '3'
        option advanced_mode '0'
        option music_source 'kuwo'
        option enable_flac '1'
        option replace_music_source 'lower_than_999kbps'
        option enable '1'



UnblockNeteaseMusic Node.js info:
Git HEAD version: Not Found
Core version: Not Found


Netease networking info:
< Location: http://m10.music.126.net/20220226232807/544da36da3d70d788e44ca705c91f6ed/ymusic/obj/w5zDlMODwrDDiGjCn8Ky/3057940231/3d32/93a3/4db1/0757f3a378b41e09ca2767c1abd47afa.mp3
{"code":200,"data":{"music.163.com":{"ip":["115.236.121.4","115.236.118.34","59.111.19.33","59.111.160.195","39.105.63.80","182.92.170.253","118.24.63.156","47.100.127.239","193.112.159.225","39.105.175.128"],"ipv4ScoreDelay":0,"prefer":"ipv4","score":false,"ttl":300}},"msg":"OK"}

Port status:
No instance found on port 5200.
No instance found on port 5201.


Running info:
PROCD running status: inactive



Failed to check latest core version, please try again later.
Core Not Found, please download it before starting.


Log is available at:
https://litter.catbox.moe/jay4d4.txt
https://transfer.sh/nGJgfG/unm-debugging-output.txt

**Comment by 1715173329** [comment, marked as answer]
梯子烂了。

**Reply by keeno1982** [reply]
哦，了解了，看起来是核心下不下来，我换个网试试

**Reply by keeno1982** [reply]
换了梯子也是不行，奇怪了。最后解决方法是拿了之前树莓派编译的系统把core相关文件复制到N1的openwrt里面。但是不能点更新，点了就完犊子。

**Comment by zxy678** [comment]
Failed to check latest core version, please try again later.
Core Not Found, please download it before starting.
我也有这个问题，后来发现是我这wget不支持https下载，把https的相关组件安装后再更新核心就能下载成功了
